src/main/scala/DemoFortis.scala

```diff
@@ -173,6 +181,66 @@ object DemoFortis {
       }
     }
 
+    if (mode.contains("tadaweb")) {
```

Comment:
These demos are getting quite many. We may want to split them out into separate classes.